src/jsonb_utils.c

```diff
@@ -171,6 +171,24 @@ ts_jsonb_get_time_field(const Jsonb *jsonb, const char *key, bool *field_found)
 	return DatumGetTimestampTz(time_datum);
 }
 
+bool
+ts_jsonb_get_bool_field(const Jsonb *json, const char *key, bool *field_found)
```

Comment:
This is unused now. Will this be used by compression policy?

Reply:
yes